### ISSUE TITLE
[Bug]: critic/value_clip_ratio is always 0 (compares the post-clip update against the clip bound)

### ISSUE THREAD

**Issue by kakiuwang-ui** [RLinf/RLinf#1529]
### Affected component

`rlinf/algorithms/losses.py` — the `critic/value_clip_ratio` metric reported by
`compute_ppo_critic_loss`, which is shared by the `actor_critic` and
`decoupled_actor_critic` loss types.

Metrics only: the value loss itself is unaffected.

### Bug description

`critic/value_clip_ratio` is pinned at `0.0` on every run. It decides whether an update was
clipped by looking at the **already-clipped** prediction:

https://github.com/RLinf/RLinf/blob/main/rlinf/algorithms/losses.py#L353-L364

```python
value_pred_clipped = prev_values + (values - prev_values).clamp(
    -value_clip, value_clip
)
...
value_clip_indicator = (value_pred_clipped - prev_values).abs() > value_clip
value_clip_ratio = value_clip_indicator.float().mean()
```

`value_pred_clipped - prev_values` *is* `(values - prev_values).clamp(-value_clip, value_clip)`
by construction, so its absolute value is `<= value_clip` for every element. The strict `>`
comparison can therefore never be true, and the metric is identically zero no matter how far
the value head moves. The quantity that is actually being asked about is the **pre-clip**
update, `values - prev_values`.

The docs describe it as a live diagnostic, which is what makes this look unintended:

`docs/source-en/rst_source/reference/metrics.rst`

```
* - ``train/critic/value_clip_ratio``
  - Fraction of value targets whose update was clipped.
```

(`docs/source-zh/.../metrics.rst`: 价值目标更新被裁剪的比例。)

A second, smaller problem in the same two lines: the mean is taken over **all** elements and
ignores `loss_mask`, so padded timesteps dilute the ratio. Both `critic/value_loss` (via
`loss_agg_func`) and `critic/explained_variance` (via
`compute_critic_explained_variance_stats`) already respect `loss_mask` in this same function,
so the three critic metrics are not computed over the same population.

### Configuration YAML file

Not configuration-specific — it applies to every config that selects a critic loss. The metric
is emitted whenever `algorithm.loss_type` is `actor_critic` or `decoupled_actor_critic`, which
is 106 of the shipped configs under `examples/` and `tests/`, e.g.
`examples/embodiment/config/libero_spatial_ppo_openpi_quickstart.yaml`:

```yaml
algorithm:
  loss_type: actor_critic
  value_clip: 0.2
  huber_delta: 10.0
```

### Log file

Deterministic CPU reproduction on current `main` (`66604ba9`) — no GPU or training run needed:

```python
import torch

from rlinf.algorithms.losses import compute_ppo_critic_loss

value_clip = 0.2
prev_values = torch.zeros(4, 8)
returns = torch.randn(4, 8)

for scale in [0.0, 0.1, 0.2, 0.5, 1.0, 10.0]:
    values = torch.full((4, 8), scale)
    _, m = compute_ppo_critic_loss(
        values=values, returns=returns, prev_values=prev_values,
        value_clip=value_clip, huber_delta=10.0, loss_mask=None,
    )
    true_frac = ((values - prev_values).abs() > value_clip).float().mean()
    print(f"{scale:>5} | reported {m['critic/value_clip_ratio'].item():.6f}"
          f" | true {true_frac.item():.6f}")
```

Actual output:

```text
  0.0 | reported 0.000000 | true 0.000000
  0.1 | reported 0.000000 | true 0.000000
  0.2 | reported 0.000000 | true 0.000000
  0.5 | reported 0.000000 | true 1.000000
  1.0 | reported 0.000000 | true 1.000000
 10.0 | reported 0.000000 | true 1.000000
```

The reported value never leaves `0.0`, including the rows where every single element is clipped.

For the masking half, with only the first 2 of 8 timesteps valid and every valid timestep
outside the trust region:

```text
reported value_clip_ratio : 0.0
true masked fraction      : 1.0
unmasked fraction         : 0.25
```

### Environment

- RLinf `main` @ `66604ba9`
- Python 3.14.6, `torch 2.13.0` (CPU)
- No accelerator involved — this is plain tensor arithmetic in the metric expression, so it is
  not platform- or backend-specific.

### Other reproduction info

Suggested fix: take the indicator from the pre-clip update and honour `loss_mask`, matching
what `compute_critic_explained_variance_stats` already does a few lines below:

```python
value_clip_indicator = ((values - prev_values).detach().abs() > value_clip).float()
if loss_mask is None:
    value_clip_ratio = value_clip_indicator.mean()
else:
    clip_mask = loss_mask.to(device=value_clip_indicator.device, dtype=torch.bool)
    if clip_mask.shape != value_clip_indicator.shape:
        clip_mask = torch.broadcast_to(clip_mask, value_clip_indicator.shape)
    value_clip_ratio = masked_mean(value_clip_indicator, clip_mask)
```

With that change the same script reports `1.000000` for the `0.5 / 1.0 / 10.0` rows and
`0.000000` for the rows inside the trust region, and the masked case reports `1.0`. The value
loss is bit-identical before and after.

I searched open and closed issues and PRs in `RLinf/RLinf` for `value_clip_ratio`,
`value_clip`, and `value_pred_clipped` and did not find an existing report. #1247
(`explained_variance` micro-batch granularity) concerns a different critic metric.

I have a focused regression test and the fix ready and will open a PR.

### Before submitting a new issue...

- [x] Have you checked relevant issues, FAQs (https://rlinf.readthedocs.io/en/latest/rst_source/resources/faq.html), or asked the chatbot at the top right corner of the [documentation page](https://rlinf.readthedocs.io).
